app/services/submit_application_choice.rb

```diff
@@ -7,11 +7,11 @@ def initialize(application_choice, apply_again: false, enough_references: false)
 
   def call
     if @apply_again && @enough_references
-      @application_choice.edit_by = Time.zone.now
+      @application_choice.application_form.update(edit_by: Time.zone.now)
```

Comment:
Here we are calling `application_form.update` up to 3 times with the ~same `edit_by` when we submit an application. This seems wrong now. Should we bubble this up to `SubmitApplication`? [There's already a convenient `update!(submitted_at` we can attach to.](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/a62645398c7abc4fd2282f3948d7f6e6014ec213/app/services/submit_application.rb#L13)

Reply:
Good spot. I've moved all the edit_by logic up to the submit application service 👍